web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -35,53 +36,62 @@
   }
 
   .recipe-sidebar {
+    background-color: $light-gray;
+    max-height: 350px;
     width: 100%;
-    overflow-y: auto;
-    height: auto;
-    margin-top: $unit * 2;
+    display: block;
+    height: fit-content;
+    padding-bottom: $unit *3;
+    margin-top: $unit *2;
 
-    @include breakpoint($desk) {
-      width: 25%;
+    &:hover,
+    &:focus {
+      color: $primary;
+      background-color: darken($light-gray, 10%);
+    }
+
+    @include breakpoint($desk-wide) {
+      width: 28%;
+      margin-left: 22px;
       margin-top: 0;
-      margin-left: $unit * 4;
     }
 
     .recipe-author-container {
```

Comment:
What's the point of this class? it's not doing anything for the styling except for adding an unnecessary level of specificity to the sass.

Reply:
You're right, it's a remnant of old code. I removed it. Thanks!